src/Indexer.js

```diff
@@ -86,6 +123,31 @@ export default class Indexer {
     return identifier
   }
 
+  /**
+   * Parses titles and subtitles out of a JSON body
+   * @param {Object} json - A Trellis resource (of some kind: RDFSource, BasicContainer, NonRDFSource, etc.)
+   * @returns {Object} an object containing title and subtitle strings if any found, null if not
+   */
+  titlesAndSubtitlesFrom(json) {
+    const titles = JSONPath({
```

Comment:
Yes, I could refactor this to make codeclimate happy. I prolly will do this, in fact, after I get one or more reviews of overall strategy.

Reply:
I *think* this method may grow into "all info sent to be indexed from a given sinopia resource", and the titles and subtitles will just be 2 pieces of the indexed info.  Would that sort of a refactor make sense?

Also wondering if we want the JSONPaths for these in this code or in a config file.  (they look pretty simple here, so config file may be unwarranted).

*and* thinking codeclimate is right - `titles` and `subtitles` are exactly the same, except for path - I think the code will be clearer with that refactoring.

Lastly, I think this method should also include the "catch-all" field of all the values in the json-ld.

Reply:
@ndushay :speech_balloon: 
> I _think_ this method may grow into "all info sent to be indexed from a given sinopia resource", and the titles and subtitles will just be 2 pieces of the indexed info. Would that sort of a refactor make sense?

Ah, yes, I like that a lot. Will do that.
 
> Also wondering if we want the JSONPaths for these in this code or in a config file. (they look pretty simple here, so config file may be unwarranted).

I'll look into pulling them into config, and see whether that makes things better or worse.
 
> _and_ thinking codeclimate is right - `titles` and `subtitles` are exactly the same, except for path - I think the code will be clearer with that refactoring.

Totally, will do that refactoring as well.
 
> Lastly, I think this method should also include the "catch-all" field of all the values in the json-ld.

Okie doke, no harm in that, and makes us less reliant on a full reindex later.

Thanks!